HARSHAL/javascript/assignment14/assignment14Q1.html

```diff
@@ -0,0 +1,49 @@
+<html>
+    <head>
+        <title>Inheritence</title>
+    </head>
+    <body>
+        <script language="javascript">
+            class student {
+                constructor(RollNo, Name) {
+                    this.RollNo = RollNo;
+                    this.Name = Name;
+                }
+                display() {
+                    document.write("<br>Inside Display");
+                    document.write("<br>Roll No " + this.RollNo);
+                    document.write("<br>Name " + this.Name);
+                }
+            }
+            //Marks is a derived class/child class
+            class Marks extends student {
+                //Construcor Overrriding
+                constructor(RollNo, Name, Marks) {
+                    super(RollNo, Name); //as we are inhereting properties of parent class we need to use super method
+                    // this.RollNo = RollNo;
```

Comment:
remove these comments

Reply:
why?

Reply:
its recommended, as the question was asked to do necessary changes so its valid to comment ?

Reply:
there is no need to push commented code

Reply:
but why ?

Reply:
poonam proper comments are expected with any code

Reply:
ok mam